contrib/containers/ci/ci.Dockerfile

```diff
@@ -65,8 +65,24 @@ RUN set -ex; \
     make install -j "$(( $(nproc) - 1 ))"; \
     cd /opt && rm -rf /opt/iwyu;
 
+# Install ctcache for clang-tidy result caching
+# Pin to specific commit to ensure patch applies correctly
+RUN set -ex; \
+    CTCACHE_COMMIT="e33c063ca6e52b48bcefbc45d46d8c150ffa81ca"; \
```

Comment:
`CTCACHE_COMMIT` should be an `ARG` like other version variables

https://github.com/dashpay/dash/blob/62f8f300c3f75561f9a2888c23c208e0ca7ff713/contrib/containers/ci/ci-slim.Dockerfile#L115